File: app/src/main/java/com/seers/servicecheck/graph_new/MeasurementType.java
package com.seers.servicecheck.graph_new;

/*
* -1은 필요 없는 항목임
* */
public enum MeasurementType {

    TEMPERATURE (30f, 44f, 35f, 38f, 38f),
    SPO2        (85f, 100f, 95f, -1f, -1f),
    BLOOD_SUGAR (0f, 200f, 100f, 140f, -1f),
    BLOOD_PRESS (0f, 140f, 80f, 120f, -1f),
    HEART_RATE  (0f, 200f, -1, 100f, -1f);

    private float minValue;
    private float maxValue;
    private float lowDangerValue;
    private float highDangerValue;
    private float textValueLocation;

    MeasurementType(float minValue, float maxValue, float lowDangerValue, float highDangerValue, float textValueLocation) {
        this.minValue = minValue;
        this.maxValue = maxValue;
        this.lowDangerValue = lowDangerValue;
        this.highDangerValue = highDangerValue;
        this.textValueLocation = textValueLocation;

    }

    public float getMinValue() {
        return minValue;
    }

    public float getMaxValue() {
        return maxValue;
    }

    public float getLowDangerValue() {
        return lowDangerValue;
    }

    public float getHighDangerValue() {
        return highDangerValue;
    }

    public float getTextValueLocation() {
        return textValueLocation;
    }
}

File: app/src/main/java/com/seers/servicecheck/graph/kt/MeasurementType.kt
package com.seers.homemonitoring.data_class

enum class MeasurementType(
    val minValue: Float,
    val maxValue: Float,
    val lowDangerValue: Float,
    val highDangerValue: Float,
    val textValueLocation: Float
) {
    TEMPERATURE(33f, 40f, 34f, 37.7f, 38f),
    SPO2(85f, 100f, 95f, -1f, -1f),
    BLOOD_SUGAR(0f, 200f, 100f, 140f, -1f),
    BLOOD_PRESS(70f, 170f, 90f, 160f, -1f),
    HEART_RATE(0f, 200f, -1f, 100f, -1f);
}